cns/configuration/configuration.go

```diff
@@ -56,7 +57,7 @@ func ReadConfig() (CNSConfig, error) {
 		//dir + string(os.PathSeparator) + defaultConfigName
```

Comment:
its not your change...can you please remove this?

Reply:
done